### ISSUE TITLE
alkyne definition - general versus specific

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#19842]
This question arose in the process of creating the terms requested in this ticket:  **L-beta-ethynylserine pathway** #17763 based on the reactions described in the recently published paper Marchand et al. as being a type of "**terminal-alkyne amino acid biosynthesis**":

Marchand JA, Neugebauer ME, Ing MC, Lin CI, Pelton JG, Chang MCY. **Discovery of a pathway for terminal-alkyne amino acid biosynthesis.** Nature. 2019;567(7748):420-424. [PMID:30867596](https://pubmed-ncbi-nlm-nih-gov.ezproxy.jax.org/30867596/) 

While Marchand et al. refers to L-beta-ethynylserine as a terminal alkyne, and in my chemistry background my recollection was that alkyne was a fairly general term encompassing a large group of organic compounds containing a triple bond, I was surprised to see this structure in ChEBI, where L-beta-ethynylserine does not classify as a type of **alkyne** and compounds classifying as an **alkyne** are limited to a very small set of compounds that meet at least one of these two formulas:  CnHn-2  &  RC#CR

### Section of ChEBI hierarchical structure
-- **acetylenic compound (CHEBI:73474)** - general formula = **C2R2**
--- **acetylenes (CHEBI:33644)**
---- acyclic acetylene (CHEBI:33650) - general formula = **C2HR**
----- **alkyne (CHEBI:22339)** - general formula = **CnH2n-2**, **RC#CR**
------ 1-undecyne (CHEBI:87545) - formula = **C11H20** 
------ 3-methyl-1-butyne (CHEBI:87379) - formula = **C5H8** 
------ acetylene (CHEBI:27518) - formula = **C2H2** _[both R groups are identical]_
------ biphenylacetylene (CHEBI:51579) - formula = **C14H10** _[matches RC#CR only & both R groups are identical]_
------ but-1-yne (CHEBI:48087) - formula = **C4H6** 
------ dec-1-yne (CHEBI:87322) - formula = **C10H18** 
------ propyne (CHEBI:48086) - formula = **C3H4** 
----- alkadiyne (CHEBI:33651)
----- alkatriene (CHEBI:33647)
----- butenyne (CHEBI:48088) - formula = **C4H4**
---- cyclic acetylenes (CHEBI:33649)
----- cycloalkyne (CHEBI:33648)
---- **terminal acetylenic compound (CHEBI:73477)**
----- _L-beta-ethynylserine (CHEBI:144833)_  - formula = C5H7NO3

### Selected ChEBI definitions
- **acetylenic compound (CHEBI:73474)** = Any organic molecule containing a C#C bond.
- **acetylenes (CHEBI:33644)** = Acyclic (branched or unbranched) and cyclic (with or without side chain) hydrocarbons having one or more carbon-carbon triple bonds.
- acyclic acetylene (CHEBI:33650) = Acyclic (branched or unbranched) hydrocarbons having one or more carbon-carbon triple bonds.
- **alkyne (CHEBI:22339)** = Acyclic branched or unbranched hydrocarbons having a carbon-carbon triple bond and the general formula CnH2n-2, RC#CR.
- **terminal acetylenic compound (CHEBI:73477)** = An acetylenic compound which a carbon of the C#C moiety is attached to a hydrogen atom.
- _L-beta-ethynylserine (CHEBI:144833)_ = A non-proteinogenic L-alpha-amino acid that is L-propargylglycine which is carrying a hydroxy group at the 3R position.

_Originally posted by @krchristie in https://github.com/geneontology/go-ontology/issues/17763#issuecomment-665769230_

**Comment by krchristie** [comment]
### IUPAC Definitions

Looking in the IUPAC GoldBook online, here are the IUPAC definitions of two relevant terms:

**alkynes**
https://doi.org/10.1351/goldbook.A00236
**Definition**: Acyclic branched or unbranched hydrocarbons having a carbon-carbon triple bond and the general formula CnH2n-2, RC≡CR. Acyclic branched or unbranched hydrocarbons having more than one triple bond are known as alkadiynes, alkatriynes, etc.
**See also**: acetylenes
**Source**: PAC, 1995, 67, 1307. (Glossary of class names of organic compounds and reactivity intermediates based on structure (IUPAC Recommendations 1995)) on page 1314 

**acetylenes**
https://doi.org/10.1351/goldbook.A00066
**Definition**: Acyclic (branched or unbranched) and cyclic (with or without side chain) hydrocarbons having one or more carbon–carbon triple bonds.
**See also**: alkynes
**Source**: PAC, 1995, 67, 1307. (Glossary of class names of organic compounds and reactivity intermediates based on structure (IUPAC Recommendations 1995)) on page 1311 

### Comparing IUPAC & ChEBI definitions

_acetylenes_
**IUPAC**: Acyclic (branched or unbranched) and cyclic (with or without side chain) hydrocarbons having one or more carbon–carbon triple bonds.
**CHEBI**: Acyclic (branched or unbranched) and cyclic (with or without side chain) hydrocarbons having one or more carbon-carbon triple bonds.

_alkyne_
**IUPAC**: Acyclic branched or unbranched hydrocarbons having a carbon-carbon triple bond and the general formula CnH2n-2, RC≡CR. _Acyclic branched or unbranched hydrocarbons having more than one triple bond are known as alkadiynes, alkatriynes, etc._
**CHEBI**: Acyclic branched or unbranched hydrocarbons having a carbon-carbon triple bond and the general formula CnH2n-2, RC#CR.

### Questions to IUPAC 
- Does the placement of the ChEBI term `alkyne` within the ChEBI hierarchy shown above meet the intended meaning of the IUPAC definition of `alkynes`?
- If the ChEBI placement of `alkyne` is not correct, where in the hierarchy should it be placed?

**Comment by krchristie** [comment]
Summary of Discussion - Ontology editors call 8/17/2020
- Response from IUPAC nomenclature experts clarifies that an `alkyne` is specifically a **hydrocarbon**, i.e. an organic compound composed only of carbon and hydrogen. Biologists and biochemists generally talk about **organic compounds**, i.e. compounds containing carbon and hydrogen, but also other types of atoms such as oxygen and nitrogen. Thus, the broader category of `acetylenic compounds` is the grouping term that encompasses the any organic molecule that contains a carbon-carbon triple bond (aka C#C). Although carbon-carbon triple bonds are relatively rare in biology, I have seen other examples where people use the term `alkyne` even what they really mean is an organic molecule that contains a C#C, i.e. an `acetylenic compound`.
- @krchristie proposed to just broaden the existing GO terms in GO for things like `alkyne metabolic process` to `acetylenic compound metabolic process` since the only child terms are a couple terms related to metabolism or catabolism of a single  alkyne, specifically acetylene. 
- @cmungall expressed a concern that this was inconsistent with guidelines that changes to a definition that broaden a term should require obsoletion. While our guidelines still do state this, it was also noted that over 10 years ago, objections were made to obsoleting terms having numerous annotations and instantiating new terms intended to cover largely the same thing. Thus since that time, we have sometimes modified a definition if it was felt this was in keeping with practicality. This may need to be discussed on a future editors call (@pgaudet ). @krchristie should make a decision about what to do in this case.
- Action items:
-- New terms will be created for `acetylenic compound metabolic process`, etc and 
-- The existing terms for `alkyne metabolic process` will be kept, with the necessary correction of the typo, from `cyclic` => `acyclic`. This is consistent with the way we represent olefins and alkenes.
-- Synonyms will be added to help biologists find the broader terms for `olefins` and `acetylenic compounds` that are more relevant in biology.

**Comment by cmungall** [comment]
great summary, thanks!

**Comment by krchristie** [comment]
In trying to implement the action item, 
- I noticed that `cellular alkene metabolic process (GO:0043449)` and `cellular alkyne metabolic process (GO:0043452)` are included in the goslim_pir set. The existing and broader (and also more recent) term for organic molecules including a C=C, `olefin metabolic process (GO:1900673)` is not included in this slim.
- In the branch where I added new terms for `acetylenic compound metabolic process` and also kept the existing terms for `alkyne metabolic process`, I am inclined to think that it is confusing and not helpful to keep the existing terms for `alkyne metabolic process`. I suspect that this may also be true for `alkene metabolic process` versus `olefin metabolic process`.

Plan:
- I will contact the keepers of goslim_pir to get input and propose to only have the more general terms since I don't think that the distinction between organic compounds versus hydrocarbons specifically is very useful to GO and it becomes confusing to users to have the more specific terms with the names that people recognize but that aren't the terms that are generally applicable to biological molecules, but only to the subset that are specifically hydrocarbons.

But this isn't an ontology development week for me, so I will get back to this next week.

**Comment by krchristie** [comment]
8/19/2020 - email from IUPAC indicates that they will be working on some clarification of IUPAC definitions in this area (alkynes, acetylenes, alkenes, olefins, etc). I don't think this will affect the major issue for GO, in that all of these terms are specific to hydrocarbons, but it should eventually clarify the structure in ChEBI in this area, which will help us too. I've also asked them to consider having official IUPAC terms for 1. any organic compound containing at least one  C=C and 2. any organic compound containing at least one C#C.

**Comment by krchristie** [comment]
- I noticed that there was a term for `cuticle hydrocarbon biosynthetic process`. Since there are cases of actual hydrocarbon metabolism, particularly in plants and insects, I decided to keep all the alkyne & alkene terms, but to add terms for hydrocarbon metabolic processes, and also modify definitions, to help make it more obvious that alkynes and alkenes are hydrocarbons.
- I added an equivalence axiom for the term `cuticle hydrocarbon biosynthetic process` and removed its asserted subClass relationships so that it now classifies automatically under `hydrocarbon biosynthetic process`.
- I noticed that there were terms for metabolism and catabolism of `halogenated hydrocarbons` and `chlorinated hydrocarbons` where the definitions were written in a way that suggested that these are subclasses of hydrocarbon. Since the addition of a halogen/chlorine atom means that the compound no longer meets the definition of a hydrocarbon, I modified these definitions to follow ChEBI.

**Comment by krchristie** [comment]
I see that there are terms for `regulation of alkane biosynthetic process`, both the positive and negative terms as well as the non-directional term. There are no annotations shown in AmiGO for any of them. The structure of the regulation branch isn't that well fleshed out with respect to the chemical hierarchy. Thus, I am not going to create any regulation terms for these new metabolic grouping terms at this time.

**Comment by krchristie** [comment]
I noticed that these two terms are included in the goslim_pir set:
* `cellular alkene metabolic process (GO:0043449)` 
* `cellular alkyne metabolic process (GO:0043452)` 

Although biologists/biochemists often refer to any organic compound containing a carbon-carbon double bond or triple bond as an alkene or alkyne, respectively, these terms are actually much narrower than that. Based on the current IUPAC definitions via ChEBI, these terms referring to `alkene` or `alkyne` are very narrowly restricted to straight chain hydrocarbons (compounds containing only carbon and hydrogen atoms) with a carbon-carbon double bound (for an alkene) or a carbon-carbon triple bond (for an alkyne).

I have recently created broader terms for any organic compound containing a carbon-carbon double or triple bond:
- olefinic compound metabolic process - The chemical reactions and pathways involving an olefinic compound, any compound which contains a carbon-carbon double bond (aka C=C).
- 


The existing and broader (and also more recent) term for any hydrocarbon containing a C=C, `olefin metabolic process (GO:1900673)` is not included in this slim and similarly to the situation with alkynes, there are no terms representing `olefinic compounds`, i.e. any organic compound containing a C=C.
